tests/framework/utils_drive.py

```diff
@@ -30,7 +30,7 @@ def partuuid_and_disk_path(rootfs_ubuntu_22, disk_path):
     initial_size = rootfs_ubuntu_22.stat().st_size + 50 * MB
     disk_path.touch()
     os.truncate(disk_path, initial_size)
-    check_output(f"echo type=83 | sfdisk {str(disk_path)}", shell=True)
+    check_output(f"echo type=83 | sfdisk --no-tell-kernel {str(disk_path)}", shell=True)
```

Comment:
Does it solve the intermittent tests?

Reply:
I don't know. As you know, it's very hard to reproduce locally. This is just the best bet. Also the `sync` is not needed by the test logic, so can be a small time saver too.